docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,174 @@
+---
+meta:
+  title: "DevTools"
+  parentTitle: "Guides"
+  description:
+    "Learn how to leverage Liveblocks DevTools to inspect and troubleshoot your
+    collaborative application."
+---
+
+Our [DevTools](/devtools) are a browser-based extension that integrates with
+Liveblocks and your local development environment. This allows you to easily
+inspect, visualize, and troubleshoot your collaborative online experiences.
+
+DevTools are currently only available if you are on Liveblocks 1.0 or later. To
+view step-by-step instructions on how to upgrade your project to Liveblocks 1.0,
+check out our [upgrade guides](/guides/upgrading/).
+
+<Banner title="Live demo video">
+
+If you want to see how this works in video, be sure to check out this
+[live demo](https://www.youtube.com/watch?v=-6Jxejp0iuw) on YouTube.
+
+</Banner>
+
+## How to use DevTools
+
+### Install the browser extension
+
+Our DevTools extension is available for all major web browsers:
+
+- [Chrome](https://chrome.google.com/webstore/detail/liveblocks-devtools/iiagocfmmhknpdalddkbiejnfmbmlffk)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/liveblocks-devtools/)
+- [Edge](https://microsoftedge.microsoft.com/addons/detail/liveblocks-devtools/hfecmmnilleegmjaegkjjklnjbgadikg)
+
+### Features and tips
+
+Within the "Liveblocks" panel, you will see two panes: storage and presence. The
+storage pane represents each object that you have defined in storage as a tree,
+likewise, the presence pane represents each user that is connected to your
+Liveblocks project.
+
+#### Highlighting
+
+When a value in storage changes, the DevTools will highlight the value to
+indicate that it has changed.
+
+<Figure>
+  <video autoPlay loop muted playsInline>
+    <source
+      src="/images/docs/devtools/highlight-changes.mp4"
```

Comment:
The video kind of blinks at the end, which looks a bit annoying

https://user-images.githubusercontent.com/10360044/222199976-fa38d01b-01ae-4ef6-817b-8e9b5290c865.mp4

Reply:
They all do